website/docs/Pipelines/SlideSeq_Pipeline/README.md

```diff
@@ -0,0 +1,234 @@
+<!--- comment about count matrix overview? --->
+
+#### 5. Merging the Star outputs into NPY and NPZ arrays
+
+The STARsolo output includes a features, barcodes, and matrix TSV for each of the partitioned FASTQ input files. The [MergeStarOutput task](https://github.com/broadinstitute/warp/blob/master/tasks/skylab/StarAlign.wdl) merges each respective TSV. It uses a custom python script to convert the merged matrix, features, and barcodes output from STARsolo into an NPY (features and barcodes)- and NPZ (the matrix)-formatted file.
+
+#### 6. Merging counts and metrics data into Loom-formatted matrix
+
+The [SlideSeqLoomOutput](https://github.com/broadinstitute/warp/blob/master/tasks/skylab/LoomUtils.wdl) task uses a custom python script to merge the converted STARsolo count matrix and the bead and gene metrics into a Loom-formatted bead-by-gene matrix. **These counts are raw and unfiltered.**
+
+<!--- comment about loom matrix overview doc --->
+
+**Spatial coordinates**
+
+The x-y coordinates of the beads are stored as a column attribute (`cell_location`) in the output Loom file.
+
+**Gene counts**
+
+The type of gene counts in the Loom will vary depending on the value of the Slide-seq workflow input, `count_exons`. By default, `count_exons` is set to false and the output Loom will contain whole-gene counts. 
+
+If the workflow is run with `count_exons` set to true, an additional layer will be created in the output Loom file containing exon counts. Using the `count_exons` parameter will cause the Loom matrix to have additional columns (bead barcodes) due to the difference in STARsolo counting mode.
+
+You can determine which type of counts are in the Loom by looking at the global attribute `expression_data_type`.
+
+After running the pipeline with `count_exons` set to true, you can access whole-gene and exonic counts using Loompy's `layers()` method. For example, `loompy.connect.layers[“”]` will return the whole-gene counts from the output Loom file. Similarly, `loompy.connect.layers[“exon_counts”]` will return the exonic counts from the output Loom. 
+
+
+#### 6. Outputs
+
+Output files of the pipeline include:
+
+1. Bead x Gene unnormalized count matrices in Loom format.
+2. Unfiltered, sorted BAM file with barcode and downstream analysis [tags](./Bam_tags.md).
+3. Bead metadata, including bead metrics.
+4. Gene metadata, including gene metrics.
+
+The following table lists the output files produced from the pipeline. For samples that have sequenced over multiple lanes, the pipeline will output one merged version of each listed file.
+
+| Output Name | Filename, if applicable | Output Type | Output Format |
+| ------ | ------ | ------ | ------ |
+| pipeline_version | N/A | Version of the processing pipeline run on this data. | String |
+| bam | `<input_id>.bam` | Aligned BAM | BAM |
+| matrix | --- | Converted sparse matrix file from the MergeStarOutputs task. | NPZ |
```

Comment:
for the missing file name values, can you find from the code what they will be named?

Reply:
Updated these